serverless-sample/app/azure/Makefile

```diff
@@ -0,0 +1,15 @@
+ifndef ACR_REPO
+    $(error ACR_REPO is undefined)
+endif
+ifndef tag
+    tag=latest
+endif
+build:
+	CGO_ENABLED=0 GOOS=linux go build -v -installsuffix nocgo -o .builds/binary-azure main.go
```

Comment:
add a az acr login command to autologin into acr

Reply:
done, added to readme